docs/deployment.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Deployment and CI/CD Process
+---
+
+# Deployment and CI/CD Process
+
+## Overview
+
+DEV relies on GitHub + Travis to continuous deploy to Heroku. If a Pull Request is merged without the "ci skip" in it's title, it will be automatically deployed to production once the test suite passes. The process currently takes about 20 minute to complete and will need a few addtional minute before the change live.
```

Comment:
`DEV relies on GitHub + Travis to continuous deploy to Heroku` -> `DEV relies on GitHub and Travis to deploy continuously to Heroku`
`in it's title` -> `in its title`
`once the test suite passes` -> `once the build steps complete succesfully` (after all it's not just the tests)
`20 minute` -> `20 minutes`
`addtional minute` -> `additional minutes`
`before the change live` -> `before the change goes live`